ArduPlane/events.cpp

```diff
@@ -172,6 +172,10 @@ void Plane::handle_battery_failsafe(const char *type_str, const int8_t action)
             FALLTHROUGH;
         case Failsafe_Action_RTL:
             if (flight_stage != AP_Vehicle::FixedWing::FLIGHT_LAND && control_mode != &mode_qland && !quadplane.in_vtol_land_sequence()) {
+                if (g.rtl_autoland == 2 && plane.mission.is_best_land_sequence()) {
```

Comment:
You would also need this for the normal auto land wouldn't you?

Reply:
seemed pointless to interrupt a RTL to do a RTL and then do_land_start.

Reply:
The comment is that in the `Failsafe_Action_Land` case you would want to not restart the landing sequence if we already are landing.

Reply:
Added to failsafe_action_land and tested.